source/utilities.tex

```diff
@@ -95,6 +95,10 @@
   template<class R, class T>
     constexpr bool in_range(T t) noexcept;
 
+  // \ref{utility.underlying}, to_underlying
```

Comment:
```suggestion
  // \ref{utility.underlying}, \tcode{to_underlying}
```

Reply:
Either it should be code font, or use a space instead of underscore. Maybe "convert to underlying".

We have others that aren't in code font but aren't English, e.g. _declval_, and others that could be interpreted either way, _swap_, _exchange_, _forward/move_, and `as_const` in code font.

Reply:
Locally, I'd prefer if `declval` and `to_underlying` were in code font.

Reply:
I think we're quoting the section heading here (plus or minus), and the section heading uses `to_underlying`.